spec/rubocop/cop/rails/safe_navigation_with_blank_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::SafeNavigationWithBlank do
+  subject(:cop) { described_class.new }
+
+  context 'target_ruby_version >= 2.3', :ruby23 do
```

Comment:
Please remove the context because RuboCop Rails requires Ruby 2.3 or higher.
https://github.com/rubocop-hq/rubocop-rails/blob/v2.3.2/rubocop-rails.gemspec#L10